bin/runtime/src/lib.rs

```diff
@@ -209,7 +209,8 @@ impl frame_system::Config for Runtime {
 impl pallet_randomness_collective_flip::Config for Runtime {}
 
 parameter_types! {
-    pub const MaxAuthorities: u32 = 100;
+    // follows polkadot https://github.com/paritytech/polkadot/blob/master/runtime/polkadot/src/lib.rs#L784
```

Comment:
use permalink instead - it may disappear

Reply:
done